platform-core-module/src/main/java/com/softicar/platform/core/module/user/login/overview/UserLoginOverviewDiv.java

```diff
@@ -0,0 +1,64 @@
+package com.softicar.platform.core.module.user.login.overview;
+
+import com.softicar.platform.common.date.Day;
+import com.softicar.platform.common.date.DayTime;
+import com.softicar.platform.core.module.CoreI18n;
+import com.softicar.platform.core.module.user.login.overview.IUserLoginOverviewQuery.IRow;
+import com.softicar.platform.dom.elements.DomDiv;
+import com.softicar.platform.dom.elements.bar.DomActionBar;
+import com.softicar.platform.dom.elements.button.DomButton;
+import com.softicar.platform.emf.EmfImages;
+import com.softicar.platform.emf.data.table.EmfDataTableDivBuilder;
+import com.softicar.platform.emf.data.table.IEmfDataTableCell;
+import com.softicar.platform.emf.data.table.column.handler.EmfDataTableValueBasedColumnHandler;
+
+class UserLoginOverviewDiv extends DomDiv {
+
+	private final DomDiv container;
+
+	public UserLoginOverviewDiv() {
+
+		this.container = new DomDiv();
+		appendChild(
+			new DomActionBar(
+				new DomButton()//
+					.setIcon(EmfImages.REFRESH.getResource())
+					.setLabel(CoreI18n.REFRESH)
+					.setClickCallback(this::refresh)));
+		appendChild(container);
+		refresh();
+	}
+
+	private void refresh() {
```

Comment:
I think we should instead
- call `new UserLoginOverviewTableDivBuilder().build()` in the CTOR, and assign its return value (an `IEmfDataTableDiv`) to a field `dataTableDiv`
- change this `refresh` method to simply call ``dataTableDiv.refresh()`

This way, filters and other table settings won't get lost when somebody clicks the "Refresh" button.